internal/cli/cli.go

```diff
@@ -110,3 +109,27 @@ func unmarshalIfExists(key string, v any) error {
 
 	return json.Unmarshal([]byte(data), v)
 }
+
+func setupDockerConfig() error {
+	dockerConfig, err := ecr.CreateDockerConfig(
+		viper.GetString("access-key"),
+		viper.GetString("secret-key"),
+		viper.GetString("registry"),
```

Comment:
Do we need to have more than one registry hooked into the ecr helper?

Reply:
If the user is pulling and pushing from a single registry, then no.

Do we potentially see use case to pull from one registry, combine into manifest, then push another?
If so, then we need support for multiple access and secret keys, but I don't think Drone would support nested `from_secret`
It would look like
```
settings:
  registries:
    public.ecr.aws:
      access_key:
        from_secret: public_ecr_access_key
      secret_key:
        from_secret: public_ecr_secret_key
    private.ecr.aws:
...
```

Reply:
Since I think this plugin is fated to just be image manifest stitching I don't think this is needed.